src/app/content/components/Content.tsx

```diff
@@ -50,21 +49,28 @@ const Background = styled.div`
 const ContentNotifications = styled(Notifications)`
   &&& {
     z-index: ${theme.zIndex.contentNotifications};
-    top: ${bookBannerDesktopMiniHeight + toolbarDesktopHeight}rem;
+    top: ${bookBannerDesktopMiniHeight + topbarDesktopHeight}rem;
     ${theme.breakpoints.mobile(css`
       top: ${({mobileExpanded}: {mobileExpanded: boolean}) => mobileExpanded
           ? bookBannerMobileMiniHeight + toolbarMobileExpandedHeight
-          : bookBannerMobileMiniHeight + toolbarMobileHeight
+          : bookBannerMobileMiniHeight + topbarMobileHeight
       }rem;
     `)}
   }
 `;
 
 // tslint:disable-next-line:variable-name
 const CenteredContentRow = styled.div`
+  grid-column: 1 / -1;
+  grid-row: 1;
+  justify-self: center;
+  width: 100%;
   overflow: visible; /* so sidebar position: sticky works */
   margin: 0 auto;
   max-width: ${contentWrapperMaxWidth}rem;
+  ${theme.breakpoints.mobile(css`
+    grid-column-start: 2;
+  `)}
```

Comment:
I think the grid here is breaking the layout on very small screens

![IMG_9001](https://user-images.githubusercontent.com/24514433/144675668-4c82ec3c-1b0a-497e-8307-0440858d3847.PNG)
![IMG_9003](https://user-images.githubusercontent.com/24514433/144675692-0dd32d16-7f17-4926-9df1-86c134875957.PNG)